### PR TITLE
Instructions to run bundle from a catalog source

### PR DESCRIPTION
**Pull request by mariomac**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,10 +56,11 @@ This task should be automatically done by the CI/CD pipeline. However, if you wa
 bundle for local testing, you should execute the following commands:
 
 ```
-export USERNAME=<container-registry-username>
+export USER=<container-registry-username>
```
Comment (OlivierCazade) [inline README.md:59]: Just an idea here, may be we could make USER default to netobserv in the namespace and then having default value of IMG to quay.io/$USER/network-observability-operator

That way, users would only have to overload the USER var.

**Review by OlivierCazade — APPROVED** [review]
LGTM, my only comment is more an improvement idea not related to your PR.

Thanks !

File: README.md (hunk 1 above)
Comment (mariomac) [reply README.md:59]: No strong opinion here, but I'm afraid that then we accidentally push images to the `netobserv` user in Quay.

Comment (OlivierCazade) [reply README.md:59]: I think, this is already the case if someone does a make image-push without any export.

Do we have permissions to push to this user ? I think only @jotak have enough permissions

Comment (mariomac) [reply README.md:59]: Me too (actually I pushed accidentally this morning 😓 ) but if it's only me, it's fine what you say.

Comment (jotak) [reply README.md:59]: Actually I'd like to keep admin rights on quay but remove write rights to prevent accidental push, I'll check if this is possible

**Pull request merged**